code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,293 @@
 			new /obj/item/clothing/mask/luchador(src)
 		if(100)
 			new /obj/item/clothing/head/bearpelt(src)
+		if(101 to 105)
+			var/ctype = pick(subtypesof(/datum/supply_pack))
+			var/datum/supply_pack/pack = new ctype()
+			for(var/item as() in pack.contains)
+				new item(src)
+			qdel(pack)
```

Comment:
Kind of makes sense, but some supply packs don't have any items, and what about illegals?

Reply:
This means sometimes abandoned crate will spawn empty I guess (and thats fine).